packages/shared/components/Alert.svelte

```diff
@@ -2,9 +2,9 @@
     import { Icon, Text } from 'shared/components'
     import { FontWeight, TextType } from 'shared/components/Text.svelte'
 
-    export let classes: string
     export let type: string
     export let message: string
```

Comment:
Can we initialize the other strings as well?

Reply:
those are required for the alert. 
The goal with adding default values to the optional ones is, that if we see a warning in the dev-console, it actually helps us because we forgot to add a required field.